CHANGES.rdoc

```diff
@@ -3,6 +3,10 @@
 This document describes the relevant changes between releases of the
 _hawkular-client_ project.
 
+=== V 0.3.0 (not released yet)
```

Comment:
It's easier to keep the track of the changes this way, but if we want to be super correct, it hasn't been released yet, so it should not be in the `CHANGES.rdoc`.

If you are asking about the 'All the clients have been unified into one', that is a left over from previous work and isn't logically related to this PR.

It's gone.